modules/layers/src/contour-layer/contour-layer.js

```diff
@@ -147,12 +146,23 @@ export default class ContourLayer extends CompositeLayer {
     let color = DEFAULT_COLOR;
     contours.forEach(data => {
       if (data.threshold === segment.threshold) {
-        color = data.color;
+        color = data.color || DEFAULT_COLOR;
       }
     });
     return color;
   }
 
+  onGetSublayerStrokeWidth(segment) {
+    const {contours} = this.props;
+    let strokeWidth = DEFAULT_STROKE_WIDTH;
+    contours.forEach(data => {
```

Comment:
Nits: 

* `contour` instead of `data`
* Use ternary operator?

`contours.forEach(contour => contour.threshold === segment.threshold ? ...`

It seems a little expensive to search all contours for each segment, but i suppose it is not a problem in practice. Still, maybe make a comment about it above the function.